buildpack.toml

```diff
@@ -71,6 +71,12 @@ api = "0.7"
     description = "colon separated list of glob patterns, matched source files are removed"
     name = "BP_EXCLUDE_FILES"
 
+  [[metadata.configurations]]
+    build = true
+    default = "false"
+    description = "whether to install Yarn/Node binaries based on the presence of a package.json or yarn.lock file"
+    name = "BP_JAVA_INSTALL_NODE"
+
```

Comment:
Should we add `BP_NODE_PROJECT_PATH`? This buildpack is using that value, it would probably make sense to add with a default of ``. 

I'd say we could also add it to the env variable list in README.md.